### PR TITLE
Barkha_Assignment_8_Done

### PR DESCRIPTION
**Pull request by BarkhaPJun20**
Create a Banking System which has following functionality.
Create two object to perform below scenario.
1) Method to debit amount
2) Method to credit amount
3) Method to printBalance [it will print current balance]
4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
Hint : Create 3 non static instance variable.
Output:
user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
Hint : Create 3 static variable will be required.
Output:
All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time 
"Assignment 8 Done"

### CODE REVIEW

File: src/barkhaPJun20/Bank.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+package barkhaPJun20;
+
+public class Bank {
+	int debitCount=0;
+	int creditCount=0;
+	int balanceCount=0;
+	
+	static int overallDebitCount=0;
+	static int overallCreditCount=0;
+	static int overallBalanceCount=0;
```
Comment (KrishnaKTechnocredits) [inline src/barkhaPJun20/Bank.java]: Remove the initialization of all instance variables and remove extra space.

File: src/barkhaPJun20/Bank.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	
+	int debitAmnt;
+	int creditAmnt;
+	int currentBalance;
+
```
Comment (KrishnaKTechnocredits) [inline src/barkhaPJun20/Bank.java]: Remove all extra space in code.

File: src/barkhaPJun20/Bank.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+	int user;
+	
+	void setInitialAmnt(int uID,int Amnt){
+		user=uID;
+		currentBalance=Amnt;
+	}
+	
+	void debitAmount( int dAmnt){
+		debitAmnt=dAmnt;
+		currentBalance=currentBalance-debitAmnt;
```
Comment (KrishnaKTechnocredits) [inline src/barkhaPJun20/Bank.java]: debitAmnt=dAmnt; This line is not needed. You should directly use dAmnt variable.
currentBalance=currentBalance-dAmnt;

File: src/barkhaPJun20/Bank.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+		debitCount++;
+		overallDebitCount++;
+	}	
+	void creditAmount( int cAmnt){
+		creditAmnt=cAmnt;
+		currentBalance=currentBalance+creditAmnt;
```
Comment (KrishnaKTechnocredits) [inline src/barkhaPJun20/Bank.java]: Same as above. creditAmnt is not required.

File: src/barkhaPJun20/Bank.java [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+	int debitCount=0;
+	int creditCount=0;
+	int balanceCount=0;
+	
+	static int overallDebitCount=0;
+	static int overallCreditCount=0;
+	static int overallBalanceCount=0;
+	
+	int debitAmnt;
+	int creditAmnt;
```
Comment (KrishnaKTechnocredits) [inline src/barkhaPJun20/Bank.java]: debitAmnt and creditAmnt both variables not needed.

File: src/barkhaPJun20/Bank.java (hunk 1 above)
Comment (BarkhaPJun20) [reply src/barkhaPJun20/Bank.java]: Done

File: src/barkhaPJun20/Bank.java (hunk 2 above)
Comment (BarkhaPJun20) [reply src/barkhaPJun20/Bank.java]: Done

File: src/barkhaPJun20/Bank.java (hunk 3 above)
Comment (BarkhaPJun20) [reply src/barkhaPJun20/Bank.java]: done

File: src/barkhaPJun20/Bank.java (hunk 4 above)
Comment (BarkhaPJun20) [reply src/barkhaPJun20/Bank.java]: Done

File: src/barkhaPJun20/Bank.java (hunk 5 above)
Comment (BarkhaPJun20) [reply src/barkhaPJun20/Bank.java]: Done

File: src/barkhaPJun20/Bank.java [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+	static int overallCreditCount;
+	static int overallBalanceCount;
+	int currentBalance;
+	int user;
+	void setInitialAmnt(int uID,int Amnt){
+		user=uID;
+		currentBalance=Amnt;
+	}
+	void debitAmount( int dAmnt){
+		currentBalance=currentBalance-dAmnt;
```
Comment (KrishnaKTechnocredits) [inline src/barkhaPJun20/Bank.java:17]: You should write the condition to verify the insufficient balance.

**Pull request merged**